### ISSUE TITLE
Mysterious crash with SpriteFinder

### ISSUE THREAD

**Issue by bleachlolxd** [Fabricators-of-Create/Create#461]
### Describe the Bug

Joining a world (singleplayer or multiplayer) with Indium and Create enabled causes visual bugs and crashes once interacted with.

Areas (not limited to entire chunks) do not render/render weirdly when they include any casing blocks
Upon attempting to break/place any casing blocks, the game crashes, while the server and nearby players are unaffected.

### Reproduction Steps

1. ~~Launch 1.18.2 with Indium 1.0.6 and Create 0.5.0c~~ Mod combination unclear now
2. Join world/Server
3. Interact with Casing Block

### Expected Result

~~Normal functionality of casing block, decoration/casing cogs and shafts.~~
Game crashes instantly, server and nearby clients are unaffected.

### Screenshots and Videos

https://www.youtube.com/watch?v=vx1mwXBvwWA
![clarification](https://user-images.githubusercontent.com/40633055/181144570-790a20c5-7bf6-4b76-ac1d-fa12f5b4b4af.png)

### Crash Report or Log

[Crash log](https://pastebin.com/E08FjECC) ~~I think, I added this after so it might be a different crash. but still same issue~~
[Full log](https://www.mediafire.com/file/cscoj7le6ybume0/2022-07-26-7.log/file)

### Operating System

Windows 10

### Mod Version

0.5.0c

### Minecraft Version

1.18.2

### Other Mods

![modlist](https://user-images.githubusercontent.com/40633055/181144561-a0670cd9-a442-4b5c-a91b-98d7bd22c0f2.png)

### Additional Context

Inconsistent at times, it could be some config I've changed, but I don't think that's the the case afaik. 
Only configs I've altered out of the given mods are 
Distant Horizons - this happens with and without it, just clarifying here first
C2ME - altered the hard chunk cache limit to save time when mass generating chunks on my server
BCLib - disabled the patcher because it makes my server take five times as long to launch

The same issue arises without the extra content mods like Adorn, AE2, Diggus, Enderscape, etc.
I only used such an extensive mod list to show the difference is only Indium in this case (compared to LunalaKT's list)

P.S. Ignore the singleplayer world name, sorry, I didn't notice until after recording what I had named it (just random letters)

**Comment by PepperCode1** [comment]
This is the same issue as PepperCode1/Continuity#121, but since you do not have Continuity it confirms that it is not directly Create's nor Continuity's fault. Some people have tried to find why this issue occurs, but no one has found a reliable reproduction case yet.

#### Some Other Information

`SpriteFinder` is a class that is part of the universal FRAPI implementation, so the fix would likely need to be applied to Fabric API to resolve the issue.

DashLoader and JEI are possible candidates for causing this issue, but since the reliable reproduction case has not been found, these mods are not confirmed to be causing the issue.

**Comment by apple502j** [comment]
@bleachlolxd Can you use [the debug build of Fabric API](https://github.com/apple502j/fabric/actions/runs/2724402635), make it crash, then send the crash report?

Also, can you reproduce this without DashLoader?

**Comment by bleachlolxd** [comment]
### Using Version of Fabric API as Seen in OP
I couldn't reproduce it without DashLoader (with Indium)
I couldn't reproduce it without Indium (with DashLoader)

I couldn't reproduce it with Create, Dashloader, Fabric API, Indium, and Sodium together on a different MMC instance
I couldn't reproduce it with the same mod list as shown in my op when on a separate instance **at all**

After duplicating the original instance, I reproduced it with all initial mods enabled
I couldn't reproduce it with Create, Dashloader, Fabric API, Indium, and Sodium together alone (on Dupe)
I couldn't reproduce it without DashLoader (with Indium) (Dupe)
I couldn't reproduce it without Indium (with DashLoader) (Dupe)

I can zip up and send my MMC instance(s) for testing if requested, since clearly they're different somehow

### Edit 3:
No one saw a thing :P
[Here's that crashlog](https://pastebin.com/QC0Z4h6A) with the debug build.
[Here's the full log](https://paste.ee/p/oj66L) if you want that too, I'm not sure which is more useful.

**Comment by apple502j** [comment]
Hmm, none of the logs from the debug build is actually there. Gonna change the debug code to log to logger (not crash report) I guess.

**Comment by bleachlolxd** [comment]
You're sure I didn't mess something up on my end? I doubt you saw my reply before the 4 or so edits but I had no idea how to use the link you sent until I figured it out half an hour later. Let me know if I did something wrong, I'll try help as much as I can.

**Comment by UberMorgott** [comment]
Hi! How did you run forge version "create 0.5" on fabric?

**Comment by RedWideWeb** [comment]
> Hi! How did you run forge version "create 0.5" on fabric?

https://youtu.be/jnD_EGRbdOc

**Comment by UberMorgott** [comment]
> > Привет! Как вы запускали версию forge «create 0.5» на ткани?
> 
> https://youtu.be/jnD_EGRbdOc

thank you so much!!!

**Comment by bleachlolxd** [comment]
The issue seems totally gone from my side using the most recent build (as of 7/30/22-04:53) found [here](https://github.com/Fabricators-of-Create/Create/actions/runs/2758284828), it's the only new version I've tried since starting this issue so this might've been fixed forever ago tbh.

**Comment by bleachlolxd** [comment]
After some playtime, it randomly came back... and I tried to get some reproduction method but I seriously have no clue with the way it acts.
"Randomly" isn't used lightly, I've never experienced a bug this wacky and hard to reproduce.
I'm genuinely stumped as to why it was gone for hours over multiple relogs/relaunches and then suddenly it's back.

[Here's](https://www.youtube.com/watch?v=md8yon6foBs) an unnecessarily long recording of me showcasing different instances with the same mods inconsistently producing the bug. (May be low quality, I uploaded it just before reopening this)

[Here's](https://pastebin.com/18LvR1gf) a crashlog from the "1.18.2 W the Boys Test2" instance shown in that video
[Here's](https://paste.ee/p/a6063) a full log from that crash.

Not sure what else to do, guess I'm just hoping this gets some fix soon so I can get back to playing with my bros.

I'm unsure whether this bug should be here/if there's a better place to put it, and if I should bother with continuing to report on it if it's being worked on already.
